templates/customize-class.html

```diff
@@ -6,7 +6,14 @@
 </button>
 <br><br>
 <div class="flex flex-col gap-2">
-    <h2>{{_('customize_class')}}: {{class_info.name}}</h2>
+    <div class="flex flex-row gap-4 items-center">
+        <h2>{{_('customize_class')}}:</h2>
+        <h2 class="mt-0">
```

Comment:
Instead of making the name of the class an input field, I would like if instead we put a pencil on the side of ne name, like 

```suggestion
        <h2>{{_('customize_class')}}: {{class_info.name}} <i class="fa-solid fa-pencil"></i></h2>
```

And when you click on it, you are greeted with a modal or the h2 turns into an input field, whatever works best for you!  Right now, the behavior that I found was that after editing the field, a modals shows up, and then there's an error, I think we should keep one behavior or the other, but not both. See the attached video


https://github.com/user-attachments/assets/89b1b822-0c28-4b87-aa5b-d4c627dd84c4


If you need to show the modal when clicking a button you can use this function: `prompt` from  `modal.ts` And you can see an example of this being used in the function `create_class` in the file `teachers.ts`

Reply:
So uppon further review, I saw that `rename_class` already uses the `modal.prompt`, so instead of hooking it up to the change event of the `input` just call it when someone press the pencil button I suggested earlier. And instead of pasing it `this.value` you are going to use a value like:  "What's the new name of this class" or something of the sorts, then I'll deal with making that value translatable.

Reply:
hmm, I think there's is something wrong with my environment, because I wasn't getting that modal showing up after the input edit, but I've changed it to be the pencil button now. It works for me but my environment may not be set up right, let me know if it works for you. For the prompt I just used the value you said here, as there is no message in the translations corresponding to rename class